package.json

```diff
@@ -7,11 +7,14 @@
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.21.1",
+    "bootstrap": "^5.0.2",
     "moment": "^2.29.1",
     "react": "^17.0.2",
+    "react-bootstrap": "^1.6.1",
     "react-datepicker": "^4.1.1",
     "react-dom": "^17.0.2",
     "react-icons": "^4.2.0",
+    "react-p5-wrapper": "^2.4.1",
```

Comment:
do we need this dependancy?

Reply:
Not sure, don't know how it got in there. Maybe this was something your Meeting Page needed?